client/src/components/TabBody/style.css

```diff
@@ -0,0 +1,29 @@
+.tabBody-guidelines {
+  text-align: right;
+  font-size: .5em;
+}
+
+.tabBody-guidelines-href {
+  color: #E60085;
+  font-weight: 600;
+}
+
+.tabBody-guidelines-href:hover {
+  color: #08A500;
+}
+
+.tabBody-question {
+  color: #000000;
+  font-size: 1em;
```

Comment:
you already put the question in h1 
so why are the color and font-size and weight

Reply:
if you put it in h1 it's already have a style and you can change the color
we are making a style for all h1 and h2